### PR TITLE
Genome-wide alternative splicing workflow

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.cwl [hunk 1]
```diff
@@ -0,0 +1,1232 @@
+#!/usr/bin/env cwl-runner
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.cwl:1]: I think you added this accidentally ? Could you remove this please

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 2]
```diff
@@ -0,0 +1,2648 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Training workflow",
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "0000-0002-5752-2155",
+            "name": "Crist\u00f3bal Gallardo Alba"
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:8]: I think something went wrong here, unicode characters should be fine here.

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 3]
```diff
@@ -0,0 +1,2648 @@
...
+        }
+    ],
+    "format-version": "0.1",
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:11]: release is missing here

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 4]
```diff
@@ -0,0 +1,2648 @@
...
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "211faabc-73a3-421c-8baa-081b359af0b3",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "2": {
+            "annotation": "Genome reference in FASTA format.",
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:70]: Is there a reason you didn't use a cached genome here ?

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 5]
```diff
@@ -0,0 +1,2648 @@
...
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "adf1c5f8-fa96-40dc-8486-28afa12a0b46",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "3": {
+            "annotation": "IDs of control sequences, corresponding to filenames.",
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:97]: I think you need to write a bit more about this. What are these ids, where to get them from etc.

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 6]
```diff
@@ -0,0 +1,2648 @@
...
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "0573145b-6868-4cd9-afbf-d07a689e9b84",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "4": {
+            "annotation": "First factor name.",
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:124]: Same here, you'll have to explain what these factors are

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 7]
```diff
@@ -0,0 +1,2648 @@
...
+            "tool_id": null,
+            "tool_state": "{\"parameter_type\": \"text\", \"optional\": false}",
+            "tool_version": null,
+            "type": "parameter_input",
+            "uuid": "ff46ec19-ed7c-4aed-8b8a-9fb0534afdf5",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "6": {
+            "annotation": "Galaxy compatible CPAT header.",
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:178]: Same here, what's a Galaxy compatible CPAT header and where to get it from ?

File: workflows/transcriptomics/alternative-splicing/alternative-splicing.ga [hunk 8]
```diff
@@ -0,0 +1,2648 @@
...
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "1ee3e8e7-96d2-45c6-83da-dec97f6b99de",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "7": {
+            "annotation": "Active sites dataset.",
```
Comment (mvdbeek) [inline workflows/transcriptomics/alternative-splicing/alternative-splicing.ga:205]: What this and where can we get this from ?

**Review by mvdbeek — COMMENTED** [review]
This looks very nice but it seems to be missing some important metadata. Sorry for the late review, feel free to ping me here or in the IWC channel if I'm missing something here!

**Comment by gallardoalba** [comment]
> This looks very nice but it seems to be missing some important metadata. Sorry for the late review, feel free to ping me here or in the IWC channel if I'm missing something here!

Thanks a lot! I'll try to work on that along the week.

**Pull request closed without merging**